mteb/evaluation/evaluators/Image/Any2AnyRetrievalEvaluator.py

```diff
@@ -283,6 +283,43 @@ def load_results_file(self):
         return previous_results
 
 
+class Any2AnyPipelineRetrievalExactSearch(Any2AnyDenseRetrievalExactSearch):
```

Comment:
Do we really need to extend from `Any2AnyDenseRetrievalExactSearch`? I think most of that class in the search method you are overriding here. The other method is `load_results_file()` not sure it is necessary, or if that logic could be moved to another class or utils function that could be reused.
If we remove that parent class, we could create a simpler constructor for this class, that takes `pipeline: RetrievalPipeline` as argument, and rename `self.model` instead of `self.pipeline`.